src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -152,72 +107,40 @@ internal static void UpdateAccentColors()
 #if DEBUG
             System.Diagnostics.Debug.WriteLine("INFO | Text on accent is DARK", "System.Windows.Accent");
 #endif
-            Application.Current.Resources["TextOnAccentFillColorPrimary"] = Color.FromArgb(
-                0xFF,
-                0x00,
-                0x00,
-                0x00
-            );
-            Application.Current.Resources["TextOnAccentFillColorSecondary"] = Color.FromArgb(
-                0x80,
-                0x00,
-                0x00,
-                0x00
-            );
-            Application.Current.Resources["TextOnAccentFillColorDisabled"] = Color.FromArgb(
-                0x77,
-                0x00,
-                0x00,
-                0x00
-            );
-            Application.Current.Resources["TextOnAccentFillColorSelectedText"] = Color.FromArgb(
-                0x00,
-                0x00,
-                0x00,
-                0x00
-            );
-            Application.Current.Resources["AccentTextFillColorDisabled"] = Color.FromArgb(
-                0x5D,
-                0x00,
-                0x00,
-                0x00
-            );
+            Application.Current.Resources["TextOnAccentFillColorPrimary"] = 
```

Comment:
Where are these values from? Did you mean to use the supplied arguments?

Reply:
These were inbuilt with the system. We use the new accent colors in the resources below. These resources are not affected by accent I guess.

In that case, these should be handled when updating rest of the resources. I will check this.